### PR TITLE
Fix in-app messages on lanscape mode

### PR DESCRIPTION
**Pull request by edobreva**
The fix prevents crashes when getting In-app messages in apps that allow only landscape mode orientation.

### CODE REVIEW

File: Mixpanel/MPNotificationViewController.m [hunk 1]
```diff
@@ -597,7 +597,7 @@ @implementation MPBgRadialGradientView
 
 - (void)drawRect:(CGRect)rect
 {
-    CGPoint center = CGPointMake(160.0f, 200.0f);
+    CGPoint center = CGPointMake(self.center.x, 200.0f); //CGPointMake(160.0f, 200.0f);
```
Comment (samgreen) [inline Mixpanel/MPNotificationViewController.m]: Can you remove the commented out lines?

File: Mixpanel/MPNotificationViewController.m [hunk 2]
```diff
@@ -120,8 +120,8 @@ - (void)viewDidLoad
     [self.okayButton addTarget:self action:@selector(pressedOkay) forControlEvents:UIControlEventTouchUpInside];
     [self.closeButton addTarget:self action:@selector(pressedClose) forControlEvents:UIControlEventTouchUpInside];
 
-    UIPanGestureRecognizer *gesture = [[UIPanGestureRecognizer alloc] initWithTarget:self action:@selector(didPan:)];
-    [self.imageDragView addGestureRecognizer:gesture];
+//    UIPanGestureRecognizer *gesture = [[UIPanGestureRecognizer alloc] initWithTarget:self action:@selector(didPan:)];
```
Comment (samgreen) [inline Mixpanel/MPNotificationViewController.m]: Why was this removed?

File: Mixpanel/MPNotificationViewController.m (hunk 1 above)
Comment (edobreva) [inline Mixpanel/MPNotificationViewController.m]: It is done
Nice week! :)

**Comment by samgreen** [comment]
This is already resolved in master. Thanks!
